content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,63 @@
 # Roles and Responsibilities
 
 ## Team Organization
+Teams can have up to 5 quizzers on it. Teams can have up to 4 quizzers in the quiz at any given time.
 
 ### Quizzers
 
 ### Captains
+- Scorekeepers must be notified which quizzer is captain and co-captain before the quizzers take their seats to begin the quiz.
+- The team captain and co-captain may change during a given competition but not during a quiz.
```

Comment:
Currently, if both the designated Captain and Co-Captain are not on the stage, no one on that team can (challenge, rebut a challenge, ask for an explanation of a ruling). 

Is this what we want?